docs/resources/vkcs_kubernetes_node_group.md

```diff
@@ -45,6 +45,7 @@ The following arguments are supported:
  Changing this will force to create a new node group.
 * `volume_type` - (Optional) The volume type to load nodes from.
  Changing this will force to create a new node group.
+* `max_node_unavailable` - (Optional) The maximum number of nodes that can fail during an upgrade. The default value is 25 percent.
```

Comment:
We're used to add "**New since vX.Y.Z**" for new attributes. Lets decide our behavior for the new provider.

Reply:
please show an example of how it should look like

Reply:
And why make it so difficult? there is a changelog for that

Reply:
done